.travis.yml

```diff
@@ -134,8 +136,8 @@ script:
   - python setup.py bdist_wheel
   - pip install ./dist/*.whl
   - curl -fsSL -o coala-bears.zip $BEARS_ZIP_URL/master
-  - pip install coala-bears.zip[alldeps]
   - coala --non-interactive
+  - pip install coala-bears.zip[alldeps]
```

Comment:
No this is wrong, how can you run coala without installing coala-bears?

Reply:
Isn't it just for dependencies? I tried running with this before but it downgrades the PyYAML package to 3.xx but coala requires PyYAML package > 5.xx .
Editing dependencies is not that easy in this case as it is deeply rooted.